.circleci/config.yml

```diff
@@ -24,6 +24,8 @@ shared: &test
           export GOSS_FILES_STRATEGY=cp
           # Sleep a bit to ensure that PHP-FPM has time to start up.
           GOSS_SLEEP=15 dgoss run -e GOSS_PHP_VERSION="${PHP_VERSION}" docker-drupal-php7-fpm:sut
+          # Ensure we have wait-for-it and that it works
+          wait-for-it -t 1 localhost:9000
```

Comment:
Could/should this be a part of the goss test?

Reply:
Agree!